src/main/java/src/chapter7/question3/Main.java

```diff
@@ -0,0 +1,24 @@
+package src.chapter7.question3;
+
+public class Main {
+    public static void getDivisibility(int number) {
```

Comment:
The method name cannot start with 'get' if it doesn't return a value.
It seems to me that in this method you print the divisibility of the number, perhaps it should be name 'printDivisibility'.

Reply:
The method name has been corrected.